x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -38,7 +38,7 @@ describe('when on the hosts page', () => {
   beforeEach(() => {
     const mockedContext = createAppRootMockRenderer();
     ({ history, store, coreStart, middlewareSpy } = mockedContext);
-    render = () => mockedContext.render(<HostList />);
+    render = () => mockedContext.render(<EndpointList />);
     reactTestingLibrary.act(() => {
       history.push('/hosts');
```

Comment:
I think a lot of the test failures are a result of this, since we updated the route name, the UI router in the tests needs to think it's on `/endpoints`